### PR TITLE
Add DALRCF722DUAL unified target

### PR DESCRIPTION
**Pull request by etracer65**
Created using a real DALRCF722DUAL board.

### CODE REVIEW

File: configs/default/DALR-DALRCF722DUAL [hunk 1]
```diff
@@ -0,0 +1,124 @@
+# Betaflight / STM32F7X2 (S7X2) 4.1.0 Oct 16 2019 / 11:58:45 (c37a7c91a) MSP API: 1.42
```
Comment (mikeller) [inline configs/default/DALR-DALRCF722DUAL]: Heh, that file name. :wink:

**Comment by mikeller** [comment]
Also, checks are working. :+1:

File: configs/default/DALR-DALRCF722DUAL (hunk 1 above)
Comment (etracer65) [reply configs/default/DALR-DALRCF722DUAL]: Oops! Fixed.

File: configs/default/DALR-DALRCF722DUAL.config [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+set max7456_spi_bus = 2
+set dashboard_i2c_bus = 1
+set flash_spi_bus = 3
+set gyro_1_bustype = SPI
+set gyro_1_spibus = 1
+set gyro_1_sensor_align = CW180
+set gyro_1_align_yaw = 1800
+set gyro_2_spibus = 1
+set battery_meter = ADC
+set current_meter = ADC
```
Comment (IvoFPV) [inline configs/default/DALR-DALRCF722DUAL.config]: @etracer65 These are repeated twice.

Comment (etracer65) [reply configs/default/DALR-DALRCF722DUAL.config]: Ahhh, you're right - thanks. Fixed.

**Pull request merged**